elastic/kibana
# [Security Solution][Data Views] - Add custom DataView error

## Summary

When a rule can't find a data view, we can show a more specific error:

Before:
<img width="1077" alt="Screenshot 2022-07-18 at 13 36 34" src="https://user-images.githubusercontent.com/7609147/179503445-88bb7bf1-1e5b-4a3c-8a0d-7029281b4eca.png">


After:
<img width="1239" alt="Screenshot 2022-07-18 at 13 33 03" src="https://user-images.githubusercontent.com/7609147/179503032-30794df0-823c-4814-8e0b-2b533197d859.png">


## How to reproduce:
- Create a rule with a data view
- export rule
- create a new space
- import that rule, you should see the error
